source/extensions/filters/udp/udp_proxy/hash_policy_impl.cc

```diff
@@ -20,13 +20,34 @@ class SourceIpHashMethod : public HashPolicyImpl::HashMethod {
   }
 };
 
+class KeyHashMethod : public HashPolicyImpl::HashMethod {
+public:
+  KeyHashMethod(const std::string& key) : Key(key) {}
+  const std::string& Key;
+
+  absl::optional<uint64_t>
+  evaluate(const Network::Address::Instance& downstream_addr) const override {
+    (void)downstream_addr;
+    // ASSERT(!Key.empty());
+    if (!Key.empty()) {
+      uint64_t hash = HashUtil::xxHash64(Key);
+      return hash;
```

Comment:
Just return the result of `HashUtil::xxHash64(Key)` for brevity.

Reply:
Yes, it's better. Thanks.